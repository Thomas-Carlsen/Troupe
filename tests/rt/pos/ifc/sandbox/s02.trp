let 
   val s = 1 raisedTo {secret} 
   fun loop i = let val _ = sleep(10) in loop (i + 1) end  
in sandbox (2000, fn () => if s>0 then loop 0 else ())
end
