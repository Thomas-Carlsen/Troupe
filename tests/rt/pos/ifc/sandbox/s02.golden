2019-03-07T12:24:22.159Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
>>> Main thread finished with value: (false@{secret}%{}, ()@{secret}%{secret})@{}%{}
